node_package/src/ClientSideRenderer.ts

```
@@ -92,7 +92,8 @@
           return;
         }
 
-        if (delegateToRenderer(componentObj, props, railsContext, domNodeId, trace)) {
+        // Abort if component unmounted during awaiting the async operation
+        if (await delegateToRenderer(componentObj, props, railsContext, domNodeId, trace) || this.state === 'unmounted') {
```

ESLint

[error] Replace `await·delegateToRenderer(componentObj,·props,·railsContext,·domNodeId,·trace)·||·this.state·===·'unmounted'` with `⏎··········(await·delegateToRenderer(componentObj,·props,·railsContext,·domNodeId,·trace))·||⏎··········this.state·===·'unmounted'⏎········`

(prettier/prettier)